rosa/cli.py

```diff
@@ -51,14 +54,21 @@ def is_logged_in(aws_region=None, allowed_commands=None):
 
 
 def execute_command(command):
+    def _wait_for_command_execution(_command):
```

Comment:
I don't understand why sampler is needed here?
You execute `rosa` command, either it passed or not.
all you need to do is add `timeout` arg to: 
`res = subprocess.run(command, capture_output=True, text=True)`

Reply:
Misunderstood, thought the issue was referring the sampler. Fixed